src/components/checkbox/_checkbox.scss

```diff
@@ -38,8 +38,7 @@
     width: rem(18px);
     margin-right: .5rem;
     background-color: $ui-01;
-    border-radius: 2px;
-    border: 2px solid $ui-05;
+    border: $checkbox-border-width solid $ui-05;
```

Comment:
I think it works fine to use the checkmark as-is since this is more of a positioning issue. No issue with the actual checkmark as far as I can tell.